cmd/mr_create.go

```diff
@@ -285,17 +285,18 @@ func mrText(base, head, sourceRemote, targetRemote string, coverLetterFormat boo
 		err       error
 	)
 	remoteBase := fmt.Sprintf("%s/%s", targetRemote, base)
+	targetBase := fmt.Sprintf("%s/%s", sourceRemote, head)
```

Comment:
May I kindly suggest that we change these names?
How does `remoteBase` relates to `targetRemote` while `targetBase` relates to `sourceRemote`?
It kinda messes with my head even knowing what the code does.

`base` here points to the target branch, so `remoteBase` could be `targetRef` or `targetBase`?
`head` is the source branch, so `targetBase` could be `sourceRef` or `sourceBase`?